### ISSUE TITLE
Is the filename aligned with CT-RATE?

### ISSUE THREAD

**Issue by cnyvfang** [Tang-xiaoxiao/M3D-RAD#2]
Thank you for your valuable work.

I noticed that your test set uses the name starting with "test", which is inconsistent with CT-RATE's "valid" prefix. So I want to know, are the ID in the filenames (test set and training set) and the files in CT-RATE one-to-one corresponding? Or did you renumber the files.

Thank you again and looking forward to your reply.

**Comment by cnyvfang** [comment]
Thank you so much! Nice work!
